operations/producer/producer-operation.go

```diff
@@ -91,25 +119,33 @@ func (operation *ProducerOperation) Produce(topic string, flags ProducerFlags) {
 	if flags.Value != "" {
 		value = []byte(flags.Value)
 	} else if stdinAvailable() {
-		bytes, err := ioutil.ReadAll(os.Stdin)
-		if err != nil {
-			output.Failf("Failed to read data from the standard input: %s", err)
-		}
+		scanner := bufio.NewScanner(os.Stdin)
+		for scanner.Scan() {
```

Comment:
of course you will have to put the code for the actual sending of the message inside the for loop so that each line is send as a message

Reply:
Put inside the for loop